scripts/rollup/forks.js

```diff
@@ -142,6 +142,14 @@ const forks = Object.freeze({
         return null;
     }
   },
+
+  // React DOM uses different top level event names and supports mouse events.
+  'events/ResponderTopLevelEventTypes': (bundleType, entry) => {
```

Comment:
This is really interesting! I was not aware of the forks system. Thank you for having another look and showing me that.